theme/js/dataset-recommendations.js

```diff
@@ -20,6 +26,10 @@ function addRecos(recos) {
         recoChildContainer.appendChild(recoChildEmbed);
         recoContainer.appendChild(recoChildContainer);
     });
+
+    if (!(typeof _paq == 'undefined')) {
```

Comment:
Why not `if (_paq !== undefined)` or `if (_paq)` ?

Reply:
`if (_paq)` will crash. `if (_paq !== undefined)`, I don't remember why but I think it's bad practice vs `typeof`.

Reply:
`if (_paq !== undefined)` will crash too actually. But my original proposal would too, I fixed it.